queries/formInstances.ts

```diff
@@ -31,6 +31,36 @@ export const getFormLabel = async function (formTtl: string) {
   }
 };
 
+export const getFormPrefix = async function (formTtl: string) {
+  const q = `
+    PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+
+    SELECT DISTINCT *
+    WHERE {
+        ?s ext:prefix ?o .
+    }
+    `;
+  const store = await ttlToStore(formTtl);
+  const engine = new QueryEngine();
+  const bindingStream = await engine.queryBindings(q, {
+    sources: [store],
+  });
+
+  const defaultPrefix = 'http://data.lblod.info/form-data/instances';
+
+  const bindings = await bindingStream.toArray();
+  if (bindings.length) {
+    const binding = bindings[0].get('o');
+    if (binding) {
+      return binding.value;
+    } else {
+      return defaultPrefix;
+    }
```

Comment:
Hopefully the 'o' will never be empty, but you never know I guess. the if can be shortened using the ternary operator though!

Reply:
o can indeed still be the empty string, would be nice to require at least length 1 or something here